tests/component_tests.py

```diff
@@ -1,23 +1,44 @@
 import unittest
 import os
+import xml.etree.ElementTree as ET
+
 from settings import Settings
 from util.common import CommonUtils, skip_on_instruments
 from util.configurations import ComponentUtils
-import xml.etree.ElementTree as ET
+from abstract_test_utils import AbstractSingleTests
 
 
-class ComponentsSingleTests(unittest.TestCase):
+class ComponentsSingleTests(AbstractSingleTests):
     """
     Tests in this class will be run exactly once regardless of how many components exist.
     """
+    TOTAL_NON_INTERESTING_PVS_IN_BLOCKS = 0
 
     def setUp(self):
-        self.component_utils = ComponentUtils(Settings.config_repo_path)
+        self.utils = ComponentUtils(Settings.config_repo_path)
 
     def test_GIVEN_components_directory_THEN_it_contains_the_base_component(self):
-        self.assertIn(ComponentUtils.BASE_COMPONENT, self.component_utils.get_configurations_as_list(),
+        self.assertIn(ComponentUtils.BASE_COMPONENT, self.utils.get_configurations_as_list(),
                       "Base component was missing (should be called {})".format(ComponentUtils.BASE_COMPONENT))
 
+    def get_config_type(self):
+        """
+        This method is used for printing messages to the screen where the only difference is the word
+        configuration/component.
+        :return: component as string.
+        """
+        return "component"
+
+    def print_total_non_interesting_block_pvs(self, num_non_interesting_block_pvs):
+        """
+        This class monitors the number of non interesting block pvs discovered so far for
+        components and print it to the screen.
+        :param num_non_interesting_block_pvs: the number of non interesting block pvs for all instruments so far.
+        """
+        ComponentsSingleTests.TOTAL_NON_INTERESTING_PVS_IN_BLOCKS += num_non_interesting_block_pvs
+        print("{} non interesting component block pvs in total across all instruments".format(
+                ComponentsSingleTests.TOTAL_NON_INTERESTING_PVS_IN_BLOCKS))
```

Comment:
Could you explain why you think it makes the test output less obvious? I think the message is very clear. The point of it is to make it clear how many non interesting block pvs there are in total across all instruments, to give us an idea of the size of the problem. Otherwise, we would not know the number.